src/arrayMethodSort.js

```diff
@@ -6,6 +6,36 @@
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
Previous review comment not fixed, please add this additional function and use it like default value of compareFunction argument